pom.xml

```diff
@@ -176,6 +177,12 @@
         </dependency>
 
         <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>${commons-io.version}</version>
```

Comment:
do we really need another dependency for this? Sources of `closeQuietly(...)` is really primitive